libexec/cli/image.expand.help

```diff
@@ -4,7 +4,7 @@ USAGE: singularity [...] image.expand [expand options...] <container path>
 Grow the Singularity disk image by the given amount.
 
 EXPAND OPTIONS:
-    -s/--size   Specify a size for an operation in MiB, i.e. 1024*1024B
+    -s|--size   Specify a size for an operation in MiB, i.e. 1024*1024B
                 (default 768MiB)
```

Comment:
Does this make sense in context of squashfs? Is it only for old format, or are we allowing (somehow) images to be expanded?

Reply:
Old format.  Note that there is now an `image` command group (and that there are now "command groups").  The `image` command group is for working with legacy, writable images.  

Also, note that the writable image format is not really legacy in the sense that it is backward compatible.  You can't open an writable image in an old version of singularity, so it is not backward compatible.  But it does preserve the same behavior.